gits/terminal.py

```diff
@@ -404,8 +404,18 @@ def _cap_dim(self):
         pass
 
     def _cap_dl(self, n):
-        """Deletes ``n`` number of lines in their entirety, with the lines
-        below moving up to fill the gap that is left.
+        """Deletes ``n`` number of lines.
+
+        On the one hand the specification says that the dl capability
+        should delete ``n`` number of lines, on the other hand in the reality
+        the dl capability just scroll up the ``n`` number of lines. Notice,
+        hat the dl capability should work together with an another capability
+        that will put the cursor on the line that is going to be deleted.
+
+        For example, in test the dl capability works together with the home
```

Comment:
I think we have to introduce `dl` at the beginning of the docstring as `the dl capability` and then reference to it as `dl`. In other words, it's not necessary to repeat `the dl capability` over and over again.